mindmeld/components/dialogue.py

```diff
@@ -988,6 +1012,18 @@ def __call__(self, request, responder):
             # Fill the form with the entities in the first query
             self._initial_fill(request)
 
+        # convert json response to FormEntity objects (deserialize)
+        for i, slot in enumerate(self._local_entity_form):
+            if not isinstance(slot, FormEntity):
+                dict_slot = slot
+                slot = FormEntity(entity=dict_slot["entity"])
```

Comment:
Why dont we expand the constructor of `FormEntity ` to take in kwargs so that we dont have to do this dict assignment and setattr steps?

Reply:
Updated